### DISCUSSION TITLE
Why we put lock file into gitignore?

### DISCUSSION THREAD

**Discussion by shixin-guo** [family/connectkit#14] [Q&A] [answered]
<img width="323" alt="image" src="https://user-images.githubusercontent.com/12186221/193379367-9002f792-6861-4e15-9a9e-fc86edb8c416.png">

**Comment by lochie** [comment, marked as answer]
Not entirely sure the reason behind this, but it may be related a Vercel cache issue we were having with our monorepo that ConnectKit was originally set up in and this ignore just got left in when migrating to this repo for the open source release. 

Thanks for highlighting this, I'll get this line of code removed and the `yarn.lock` file included in the next update! ✨
